### PR TITLE
feat image html and css

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot (86)
Screenshot (87)

### CODE REVIEW

File: Styles/image.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+.responsive-image {
+    width: 100%;
+    height: auto;
+}
+
+.image-container {
+    border: 1px solid var(--border-color);
+    width: 100%;
+    height: 150px;
```
Comment (shubhamkumar648) [inline Styles/image.css:19]: you can use absolute vale rem for better responsive

**Review by shubhamkumar648 — APPROVED** [review]
your code looks clean and good to me  , you can merge

File: Styles/image.css (hunk 1 above)
Comment (aniketsingh13) [reply Styles/image.css:19]: ok will change that

**Review by SanketDhabarde — APPROVED** [review]
Looks good to me

**Pull request merged**